### PR TITLE
OSDOCS#12650: adding back docs to BM IPI postinstall config doc

### PR DESCRIPTION
**Pull request by skopacz1**
[OSDOCS-12650](https://issues.redhat.com/browse/OSDOCS-12650)

Version(s): 4.16+

This PR adds a few sections for user-managed load balancers that were deleted from bare metal IPI postinstall docs in [this PR](https://github.com/openshift/openshift-docs/pull/76524/files#diff-22f6e1dc5f61bb3bfaef5633ac8a882d54b382346a23f46bf2b2f163b082a688).

QE review:
- [x] QE has approved this change.

Preview: [Services for a user-managed load balancer](https://84830--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_bare_metal/ipi/ipi-install-post-installation-configuration.html#nw-osp-services-external-load-balancer_ipi-install-post-installation-configuration)

### CODE REVIEW

**Comment by skopacz1** [comment]
@gpei could you PTAL or tag someone who could review this? Thanks.

(For context, Dell's Apex solution needs these sections added back to the Day 2 doc to confirm the supportability of this configuration, which their automation does on Day 2. Right now the 4.16 and 4.17 release of their solution is blocked by this not being documented, and are waiting for the docs to be added back to ship these versions of their product)

**Comment by gpei** [comment]
@skopacz1 Hi, considering this is about IPI Baremetal installations, so maybe @jadhaj or @sgoveas could help to check.

File: installing/installing_bare_metal/ipi/ipi-install-post-installation-configuration.adoc [hunk 1]
```diff
@@ -8,10 +8,18 @@ toc::[]
 
 After successfully deploying an installer-provisioned cluster, consider the following postinstallation procedures.
 
+// Configuring NTP for disconnected clusters
 include::modules/ipi-install-configuring-ntp-for-disconnected-clusters.adoc[leveloffset=+1]
 
+// Enabling a provisioning network after installation
 include::modules/nw-enabling-a-provisioning-network-after-installation.adoc[leveloffset=+1]
 
+// Configuring a user-managed load balancer
```
Comment (sgoveas) [inline installing/installing_bare_metal/ipi/ipi-install-post-installation-configuration.adoc:17]: This comment should be on line 20 and line 20 comment should be here

Comment (skopacz1) [reply installing/installing_bare_metal/ipi/ipi-install-post-installation-configuration.adoc:17]: Good catch, thanks!

**Comment by skopacz1** [comment]
@sgoveas Thanks for the review, let me know if there's any other feedback I should apply

**Review by snarayan-redhat — COMMENTED** [review]
LGTM

**Comment by skopacz1** [comment]
One last push because I seem to have added these two modules in between an existing pair of modules that should have been directly next to each other, none of the content has changed besides fixing the page TOC

**Pull request merged**